<?xml version="1.0" encoding="utf-8"?>
<TestRun id="2e622c16-e858-4d1a-b9c7-5f751a43733a" name="anna@cloudtest-win10 2023-01-22 19:02:11" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T19:02:11.6795657+00:00" queuing="2023-01-22T19:02:11.6795663+00:00" start="2023-01-22T19:02:04.1283861+00:00" finish="2023-01-22T19:02:11.7188418+00:00" />
  <TestSettings name="default" id="097ce915-87a2-43d9-927d-0677b3679333">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_19_02_11" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="132608fd-a70b-4aea-b9bb-cbaa4cb9e477" testId="c38916fd-2ff6-d3fb-5c7c-1ed9cf77097b" testName="Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_ReadAll_Insert_ReadAll" computerName="cloudtest-win10" duration="00:00:01.7178003" startTime="2023-01-22T19:02:11.3648916+00:00" endTime="2023-01-22T19:02:11.3649357+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="132608fd-a70b-4aea-b9bb-cbaa4cb9e477">
      <Output>
        <ErrorInfo>
          <Message>InsertRow failed&#xD;
Expected: True&#xD;
Actual:   False</Message>
          <StackTrace>   at UnitTests.MembershipTests.MembershipTableTestsBase.MembershipTable_ReadAll_Insert_ReadAll(Boolean extendedProtocol) in C:\Users\anna\orleans\test\TesterInternal\MembershipTests\MembershipTableTestsBase.cs:line 236&#xD;
   at Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_ReadAll_Insert_ReadAll() in C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\AzureMembershipTableTests.cs:line 88&#xD;
--- End of stack trace from previous location ---</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_ReadAll_Insert_ReadAll" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="c38916fd-2ff6-d3fb-5c7c-1ed9cf77097b">
      <Execution id="132608fd-a70b-4aea-b9bb-cbaa4cb9e477" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureMembershipTableTests" name="MembershipTable_Azure_ReadAll_Insert_ReadAll" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="c38916fd-2ff6-d3fb-5c7c-1ed9cf77097b" executionId="132608fd-a70b-4aea-b9bb-cbaa4cb9e477" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" passed="0" failed="1" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.27]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.49]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.50]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:00:04.27]       InsertRow failed&#xD;
[xUnit.net 00:00:04.27]       Expected: True&#xD;
[xUnit.net 00:00:04.27]       Actual:   False&#xD;
[xUnit.net 00:00:04.27]       Stack Trace:&#xD;
[xUnit.net 00:00:04.27]         C:\Users\anna\orleans\test\TesterInternal\MembershipTests\MembershipTableTestsBase.cs(236,0): at UnitTests.MembershipTests.MembershipTableTestsBase.MembershipTable_ReadAll_Insert_ReadAll(Boolean extendedProtocol)&#xD;
[xUnit.net 00:00:04.27]         C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\AzureMembershipTableTests.cs(88,0): at Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_ReadAll_Insert_ReadAll()&#xD;
[xUnit.net 00:00:04.27]         --- End of stack trace from previous location ---&#xD;
[xUnit.net 00:00:04.30]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Min" timestamp="2023-01-22T19:02:11.3602036+00:00">
        <Text>[xUnit.net 00:00:04.26]     Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_ReadAll_Insert_ReadAll [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T19:02:11.5326394+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>